package-lock.json

```diff
@@ -17210,36 +17206,96 @@
 		"@wordpress/a11y": {
 			"version": "file:packages/a11y",
 			"requires": {
-				"@babel/runtime": "^7.11.2",
+				"@babel/runtime": "^7.12.5",
 				"@wordpress/dom-ready": "file:packages/dom-ready",
 				"@wordpress/i18n": "file:packages/i18n"
+			},
+			"dependencies": {
```

Comment:
It means that for some unknown reason, npm doesn't hoist `@babel/runtime` for all WordPress packages and will install copied for all packages that use it.

It looks like it wasn't updated in the `node_modules/@babel/runtime` to the same version.

Reply:
That's probably because in my `node_modules`, the old version of the package was already occupying the top slot. And because it's not declared in the root `package.json`, there's nothing that would force it out of there. Subpackages that need a newer version need to install a private copy.

I'll wrangle the lockfile to make it deduped again.

Reply:
In similar situations, I temporarily put the dependency in the top `package.json` and then remove it. It works like a charm. 😃

Reply:
@gziolo In the latest version of the branch, the lockfile has been updated to hoist all Babel 7.12 packages to the top, and force packages that explicitly require some older version (e.g., `remark-mdx`) to install their private copy instead of occupying the hoisted slot. I had to do some magic to achieve that, like writing a script that modifies the lockfile as needed 🙂 

The changelogs for the `babel-preset-default` and `scripts` packages were updated.

Let me know if there's anything else that needs patching.

Reply:
I'm not surprised you had to write a script as it's one of the most popular dependencies, but the number of lines removed was worth it. If only `npm` could do it instead 🙏

Reply:
`npm dedupe` would be supposed to do a big part of this job, but when running it on the repo, it ended up in an infinite loop instead 🙃